invenio_accounts/translations/messages.pot

```diff
@@ -1,22 +1,23 @@
 # Translations template for invenio-accounts.
-# Copyright (C) 2017 CERN
+# Copyright (C) 2020 CERN
+# Copyright (C) 2020 TUBITAK
 # This file is distributed under the same license as the invenio-accounts
 # project.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
```

Comment:
I think this line is not needed :)

Reply:
An automatically added line, will remove it.